ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.14-periodics.yaml

```diff
@@ -0,0 +1,73 @@
+periodics:
```

Comment:
Needs the reporter config like in [4.15 ](https://github.com/openshift/release/pull/43528/commits/ded16ca0a489536826ae8391634add09b49bf027#diff-a32d08c4317a2a719e6160fc3af4447deadaf3c041cd08ac76a62726a6962039R17)